src/_guides/language/effective-dart/style.md

```diff
@@ -420,13 +437,15 @@ Note that dartfmt does 99% of this for you, but the last 1% is you. It does not
 split long string literals to fit in 80 columns, so you have to do that
 manually.
 
-We make an exception for URIs and file paths. When those occur in comments or
-strings (usually in imports and exports), they may remain on a single line even
-if they go over the line limit. This makes it easier to search source files for
-a given path.
+**Exception:** When a URI or file path occurs in a comment or string (usually in
+an import or export), it may remain whole even if it causes the line to go over
+80 characters. This makes it easier to search source files for a path.
 
+**Exception:** Multi-line strings can contain lines longer than 80 characters
+since newlines are significant inside the string and splitting the lines into
```

Comment:
since -> because
(some people — especially non-native English speakers — have trouble distinguishing between the time sense of "since" and the causality sense.)